submissionaccesses.md

```diff
@@ -0,0 +1,48 @@
+# SubmissionAccesses Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**/api/config/submissionaccesses**   
+
+Provide access to the existent configurations for the submission access. It returns the list of existent SubmissionAccessConfig.
+The SubmissionAccessConfig aggregates in a single place the configuration needed to the access section.
+
+## Single Submission-Form 
+**/api/config/submissionaccesses/<:access-name>**
+
+*:access-name* the access configuration name to retrieve
+
+Provide detailed information about a specific input-form. The JSON response document is as follows:
+```json
+{
+  "id": "default",
+  "discoverable": true,
+  "accessConditionOptions": [
+		{
+ 			"name": "openaccess"
+		},
+		{
+ 			"name": "administrator"
+		},  	 			
+		{
+ 			"name": "embargo",
+ 			"hasStartDate": true,
+ 			"maxStartDate": "2018-06-24T00:40:54.970+0000"
+		},
+		{
+ 			"name": "lease",
+ 			"hasEndDate": true,
+ 			"maxEndDate": "2017-12-24T00:40:54.970+0000"
+		}
+  ]
+}
+
+```
+The **discoverable** flag indicates whether or not the user has the opportunity to indicate that the current item must be private, i.e. not searchable
```

Comment:
I like the change to call this "discoverable" instead of "private"!  I'd suggest simplifying this text to remove mention of "private"...say "to indicate that the current item is not findable via search or browse".

(Sidenote: I'd recommend we consider in the UI calling this "discoverable" too, instead of "private"...but that's for a different PR)